front/app/modules/commercial/verification/citizen/components/VerificationMethods.tsx

```diff
@@ -72,10 +72,10 @@ const StyledAvatar = styled(Avatar)`
 `;
 
 const ShieldIcon = styled(Icon)`
+  opacity: 0.5;
```

Comment:
We had some shield icons that were off since the upgrade, I made them look alike.